### ISSUE TITLE
Add sub-navigation to the settings screen

### ISSUE THREAD

**Issue by GBKS** [monero-project/monero-gui#1361]
![monero-wallet-settings-nav-gbks-180429](https://user-images.githubusercontent.com/695901/39406210-e01d2c0c-4b80-11e8-8cd3-02f0fdce60c4.png)

The settings screen is currently a long list of buttons and toggles that is hard to parse. I attached an idea of how we could split this screen into multiple sub-sections. This helps users ideally find the settings they are looking for more quickly (by just picking the right nav option). It also gives each section more space for explanations. For the log section, the actual log display could be within the screen (instead of being an overlay).

For mobile, four short section labels easily fit horizontally (see attached). But if there will be more, then a vertical list would work better.

This issue is just about the navigation. I tried to revise how the settings are laid out also, add some explanations, etc, but I'm sure I missed a lot of details there, so please ignore those. 

Other than that, feedback is highly appreciated and I'm happy to iterate this design as needed. Thanks.

**Comment by krtschmr** [comment]
very nice indeed. the node page is propably the most important one for beginners thus it needs more improvement. why we can't just provide a list with default trusted remote-nodes? because we trust nobody?

**Comment by GBKS** [comment]
@krtschmr that's a valid topic, how about we split that into a separate issue? I'd love to keep the topic here on just the navigation UI and the basic split-up of the options.

**Comment by skftn** [comment]
> why we can't just provide a list with default trusted remote-nodes? because we trust nobody?

Trusted by whom is the question. Monero is about trusting no one but the code. We can't have a predefined list of nodes in the official wallet, it would go against decentralization.

> a friend created his wallet today and without my help he would never been able to use a remote node. he doesn't even know what a port is.

Users want a more trivial experience while Monero developers/contributors want to maintain operational security. 

We must meet somewhere in the middle where we encourage users to run a full-node yet make it more apparent that using a remote node is an option. It would still require some manual actions, such as filling in the host/port but they will be more aware it is an option.

Wallets with a predefined remote-node list, or even a remote-node wallet only would have to be third-party.

**Comment by pazos** [comment]
@krtschmr: please give your friends a link to [GUI guide](https://github.com/erciccione/monero-GUI-guide/blob/master/monero-GUI-guide.md#212-daemon-settings). Is better to educate people than to be always fixing their stuff. :dancer:

**Comment by rbrunner7** [comment]
Definitely an improvement, IMO. I think mostly people new to Monero are drawn to the GUI wallet, and for such people the additional explanations that this layout allows are very valuable.

**Comment by lopanism** [comment]
Another piece of feedback on the wallet UI: try to simplify main menu as well. 
Note that some elements are nouns, and some verbs, this can be improved, as verbs are really actions (i.e. Send, Receive), while nouns are ‘destinations’ (i.e. Address book, settings, transactions.) 

The best place to start would be to try to separate nouns from verbs and see where it gets you. Having
Send/Receive as persistent buttons on the screen might be an interesting idea. 

Then you can categorize nouns:
Settings and Advanced are far less important ad Transactions (at a first glance), address book might not even deserve its own entity, and live contextually in Teansactions and Send/Receive flows. 

If you think along these lines, you can get to a drastically simplified navigation, and overall app structure. Which is easier to use, build and maintain. 

I can provide examples if it interests you.

**Comment by skftn** [comment]
I started working on this. Some WIP:

![](https://i.imgur.com/sKsFRum.png)

![](https://i.imgur.com/9BWkALp.png)

**Comment by GBKS** [comment]
@skftn check out the "Icons" screen in Zeplin (zpl.io/a8E3WJx), should already be there. But let me know if it's not, or if you need a different size, etc.

**Comment by mmbyday** [comment]
+resolved
by 1511

**Comment by erciccione** [comment]
+resolved

**Comment by krtschmr** [comment]
#1511
